src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1194,13 +1211,14 @@ private static Symbol GetNearestOtherSymbol(ConsList<TypeSymbol> list, TypeSymbo
             return other;
         }
 
+#nullable enable
         // Lookup member in an extension type
         private void LookupMembersInExtension(
             LookupResult current,
             NamedTypeSymbol type,
-            string name,
+            string? name,
```

Comment:
>string? name, [](http://example.com/codeflow?start=12&length=13)

I think it would be better to not use this method to collect all members for semantic model purposes. I am not confident that it is going to behave reasonably without the name.